packages/teams-js/src/public/runtime.ts

```diff
@@ -205,3 +206,27 @@ export function generateBackCompatRuntimeConfig(highestSupportedVersion: string)
 export function applyRuntimeConfig(runtimeConfig: IRuntime): void {
   runtime = deepFreeze(runtimeConfig);
 }
+
+/**
+ * @hidden
+ * Hide from docs.
+ * ------
+ * Constant used to set minimum runtime configuration
+ * while un-initializing an app in unit test case.
+ *
+ * @internal
+ */
+export const _minRuntimeConfigToUninitialize = {
```

Comment:
Seems like this PR is two changes- changing the logs/log and also changing the uninitialize structure. Maybe it would be good to split into two PRs? Or maybe that's already been discussed and decided against

Reply:
Great catch Angela. I agree that it would be cleaner to separate these into two separate PRs.